shared/message/support/Gazebo/GazeboRobotLocation.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package message.support.Gazebo;
+
+message GazeboRobotLocation {
+    double x = 1;
```

Comment:
vec3 of the form rABc

Also, can't we get orientation so we can get Htw?

Reply:
you think we know what space this is in?